providers/src/airflow/providers/google/cloud/sensors/cloud_composer.py

```diff
@@ -122,6 +123,7 @@ def poke(self, context: Context) -> bool:
         if datetime.now(end_date.tzinfo) < end_date:
             return False
 
+        self._composer_airflow_version = self._get_composer_airflow_version()
```

Comment:
Why would you need to run it on every poke? Do you expect the version to change?

Reply:
@eladkal the version won't change every poke. I have updated the code. Thank you for noticing it